core/src/test/java/apoc/cypher/CypherInitializerTest.java

```diff
@@ -29,20 +29,25 @@
 
 public class CypherInitializerTest {
 
-    public DbmsRule dbmsRule = new ImpermanentDbmsRule();
+    public DbmsRule db = new ImpermanentDbmsRule();
 
     @Rule
-    public RuleChain ruleChain = new EnvSettingRule().around(dbmsRule);
+    public RuleChain ruleChain = new EnvSettingRule().around(db);
 
     @Before
     public void waitForInitializerBeingFinished() {
         // we need at least on APOC proc being registered in finished CypherInitializers
-        TestUtil.registerProcedure(dbmsRule, Utils.class);
+        TestUtil.registerProcedure(db, Utils.class);
 
         waitForInitializerBeingFinished(SYSTEM_DATABASE_NAME);
         waitForInitializerBeingFinished(DEFAULT_DATABASE_NAME);
     }
 
+    @After
+    public void tearDown() {
```

Comment:
should this be AfterClass instead as we don't create/start the db per test method. and doing it per method would explode the test runtime.